SECURITY.md

```diff
@@ -0,0 +1,240 @@
+
+Before any embargoed information from the list is shared with respective members of your team required to fix said issue, they must agree to the same terms and
+only find out information on a need-to-know basis.
+
+In the unfortunate event that a member shares the information beyond what is allowed by this policy, that member ***must*** urgently inform the Caliptra
+workgroup at the [Vulnerabilities.caliptra-wg@lists.chipsalliance.org](mailto:Vulnerabilities.caliptra-wg@lists.chipsalliance.org) mailing list of
+exactly what information leaked and to whom. A retrospective will take place after the leak so we can assess how to not make the same mistake in the future.
+
+If the member continues to leak information and break this policy, the member will be removed from the list.
+
+More details of how vulnerabilities are handled can be found in our [Security incident management process](#security-incident-management-process)
+section.
+
+## Product creator notification process
+
+Product creators who are not already members of the Caliptra project may be eligible to participate in the
+[Vulnerability-Alerts.caliptra-wg@lists.chipsalliance.org](mailto:Vulnerability-Alerts.caliptra-wg@lists.chipsalliance.org) mail list and receive
+advanced notification of the vulnerabilities and mitigations before public disclosure by applying to participate. For more information, refer to the
+[Vulnerability notification process](#vulnerability-notification-process) section of this document.
+
+Criteria for participation includes:
+
+1. Register as a [Product creator](https://github.com/chipsalliance/Caliptra/blob/main/README.md#product-creator-registration)
+1. Have a contact who will respond to emails within a week and understands how Caliptra is being used in the product.
+1. Have a publicly listed product based on some release of Caliptra.
+1. Have an actively monitored security email alias.
+1. Accept the Caliptra Embargo Policy that is outlined above.
+
+Removal:
+
+1. If a member stops adhering to these criteria after joining the list then the member will be unsubscribed.
+
+## Security incident management process
+
+Issues within this bug tracking system will transition through a number of states according to this diagram:
+
+<center>
+
+```mermaid
+flowchart TB
+    Triage --> Draft_New
+    Triage --> Closed_Rejected
+    Draft_New --> Draft_Assigned
+    Draft_New <--> Closed_Rejected
+    Draft_Assigned <--> Draft_Review
+    Draft_Review ---> Closed_Rejected
+    Draft_Review --> Draft_Accepted
+    Draft_Accepted --> Closed_Published
+```
+
+</center>
+
+- **Triage:** This state represents new reports that have been entered directly by a reporter. When entered by the security team in response to an
+email, the issue shall be transitioned directly to *Draft_New*.
+
+- **Draft_New:** This issue is awaiting Triage by the security team. The security team will analyze the issue, determine a responsible entity, assign
```

Comment:
`Triage`-->`triage`

Reply:
reworked this description and the one above to be more clear